ttnn/cpp/ttnn/operations/conv/conv3d/device/kernels/reader_vol2col.cpp

```diff
@@ -0,0 +1,156 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+#include <algorithm>
```

Comment:
this will slow down the compile time of the kernel, if you use it just for std::min I would suggest rolling your own, not stl header is kernels is prefered.